Xamarin.Forms.Core/Popup/Popup.cs

```diff
@@ -0,0 +1,138 @@
+using System;
+using System.Threading.Tasks;
+
+namespace Xamarin.Forms
+{
+	public abstract class BasePopup : View
+	{
+		protected internal BasePopup()
+		{
+			Color = Color.White;
+			BorderColor = default(Color);
+			VerticalOptions = LayoutOptions.CenterAndExpand;
+			HorizontalOptions = LayoutOptions.CenterAndExpand;
+		}
+
+		/// <summary>
+		/// Gets or sets the <see cref="View"/> to render in the Popup.
+		/// </summary>
+		/// <remarks>
+		/// The View can be or type: <see cref="View"/>, <see cref="ContentPage"/> or <see cref="NavigationPage"/>
+		/// </remarks>
+		public virtual View View { get; set; }
+
+		/// <summary>
+		/// Gets or sets the <see cref="Color"/> of the Popup.
+		/// </summary>
+		/// <remarks>
+		/// This color sets the native background color of the <see cref="Popup"/>, which is
+		/// independent of any background color configured in the actual View.
+		/// </remarks>
+		public Color Color { get; set; }
```

Comment:
None of these properties were made bindable because you have 100% control of the `View` property where you would apply your `BindingContext`. I can see some good arguments for bindable properties on this class. I am going to update the following properties to be bindable
* **Color**
* **BorderColor**
* **Size**